mu-plugins/blocks/query-filter/render.php

```diff
@@ -38,8 +38,20 @@
 ];
 $encoded_state = wp_json_encode( [ 'wporg' => [ 'queryFilter' => $init_state ] ] );
 
+$has_multiple = isset( $attributes['multiple'] ) && $attributes['multiple'];
```

Comment:
After this change the default will be single, is that likely to be the most common configuration?

The code ends up checking the inverse in several places, eg. `'is-single-select' => ! $has_multiple,` which makes me think maybe the attribute should be `single: boolean`, thereby simplifying the logic?

Reply:
Ah no I see the default is `multiple: true`, ignore that bit

Reply:
I was thinking about it like the `select` element's `multiple` attribute, but I could change it if this seems unclear?